stdlib/2and3/numbers.pyi

```diff
@@ -70,31 +72,27 @@ class Real(Complex, SupportsFloat):
         @abstractmethod
         def __ceil__(self) -> int: ...
         @abstractmethod
-        @overload
-        def __round__(self, ndigits: None = ...): ...
-        @abstractmethod
-        @overload
-        def __round__(self, ndigits: int): ...
-    def __divmod__(self, other): ...
-    def __rdivmod__(self, other): ...
+        def __round__(self: _T, ndigits: Optional[int] = ...) -> _T: ...
```

Comment:
The docstring says "If ndigits is omitted or None, returns an Integral, otherwise returns a Real".